compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -662,7 +674,17 @@ static bool willNotInlineCompareAndSwapNative(TR::Node *node,
 #endif
    }
 
-//some recognized methods might be transformed into very simple hardcoded assembly sequence which doesn't need register spills as a real call does
+   
+/** Identify methods which are not transformed into inline assembly.
```

Comment:
Thanks for adding the Doxygen comment.  Could you use `@brief` in front of the first couple sentences?